internal/pkg/flink/internal/controller/input_controller.go

```diff
@@ -121,6 +121,9 @@ func (c *InputController) RunInteractiveInput() {
 		})
 
 		processedStatement, err = c.store.WaitPendingStatement(ctx, *processedStatement)
+		if processedStatement.StatusDetail != "" {
```

Comment:
processedStatemenent can be nil here, right?

Reply:
Good catch!